### DISCUSSION TITLE
docker compose 怎么部署？

### DISCUSSION THREAD

**Discussion by Freddy23317** [DIYgod/RSSHub#16567] [Q&A]
### What feature is it?

如题，我下载了docker desktop,下载了[docker-compose.yml](https://github.com/DIYgod/RSSHub/blob/master/docker-compose.yml)
下一步的wget https://raw.githubusercontent.com/DIYgod/RSSHub/master/docker-compose.yml这串代码去哪里运行？windows 的power shell?复制这串代码按下回车键显示' 不是内部或外部命令，也不是可运行的程序
或批处理文件。接下来怎么做？部署这一步卡了好久
我希望作者的depoy 教学能够再简单一点，对初学者友好一些

### What problem does this feature solve?

降低人们部署rss的门槛

### This is not a duplicated feature request or new RSS proposal

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) to ensure this feature has not already been requested and this is not a [new RSS proposal](https://github.com/DIYgod/RSSHub/issues/new?assignees=&labels=RSS+proposal&template=rss_request_en.yml).

**Comment by TonyRL** [comment]
https://stackoverflow.com/questions/66068581/how-to-run-docker-compose-using-docker-desktop

**Reply by Freddy23317** [reply]
vi docker-compose.yml  # 也可以是你喜欢的编辑器
这句代码输入后PowerShell 不工作![IMG_20240828_223531.jpg](https://github.com/user-attachments/assets/9f121a52-d7cc-4cf5-afba-145734d023c7)

我的电脑上只装有vs code编辑器
我试着把vi docker-compose.yml  #里的vi 换成vs vc vscode但都不起作用
我该怎么做来让我完成这些开始步骤

**Reply by TonyRL** [reply]
You can always use Windows' built-in notepad to check and edit the docker compose file.

**Reply by Freddy23317** [reply]
got it, thank you and what can i do to make PowerShell work, by changing what, vi into wb？or windows built in？

**Reply by SpriteOvO** [reply]
Try `code docker-compose.yml` for VSCode.
